### PR TITLE
Spelling fixes, made antag conversions more obvious

### LINKED ISSUE #9200: Cult deconversion text recolor
(WEB REPORT BY: termedclepe2 REMOTE: 158.69.55.2:7777)
Currently it is colored the same as regular black text without bold and easily overlooked, it should be in red and larger font for everyone to see.

### LINKED ISSUE #13574: Ejected donutstation(?) vault air alarm typo
(WEB REPORT BY: sonixapache REMOTE: 172.93.109.202:7777)
>Revision (Should be above if you're viewing this from ingame!)
05d10793eb7758240e47ff950e0ee92f2f3d11c5 

>General description of the issue
--- Atmosphere alarm detected in Engine Monitering! (Engine Monitering)

also holy shit is it annoying to have access to that area's cameras as AI but whatever

### LINKED ISSUE #15489: No error message on crowbar step of deconstruction of morgue tray
(WEB REPORT BY: valkyrieskies REMOTE: 172.93.109.202:7777)
5fce9b9dc013628e39f3f8f550d85b98a6f8ab1f 
As title, there's no message to let you know what you're doing, just a progress bar.

### LINKED ISSUE #16334: Alien Cloak Typo
https://github.com/vgstation-coders/vgstation13/blob/693c05290fc9777df57b21b622787366b814c632/maps/RandomZLevels/hive.dm#L764

>soft to the tough

### LINKED ISSUE #16408: Attempting to eject empty cryo tubes says "the sleeper is empty"

### LINKED ISSUE #16463: Admin's round panel (AKA check antagonists panel) timestamp is formatted incorrectly
`Round Duration: 1:058:08`

### LINKED ISSUE #16758: Banana Honk Reagent Called Banana Moma
(WEB REPORT BY: mask2697 REMOTE: 172.93.109.202:7777)
# Revision
5ccc46034e0b2da362b0865fb4a4d10d9df6f732

# Description

Banana Honk shows up as banana moma in the reagents
# Steps to Reproduce

Make a banana honk
# What you Expected

To get a banana honk reagent
# What Actually Happened

Got a banana mama

### LINKED ISSUE #16832: Excrutiating
Apparently it's spelled "Excruciating" who knew?
untitled
It's amazing how long it's been this way.

### LINKED ISSUE #16978: Sweet Machine typo
(WEB REPORT BY: assfaget REMOTE: 172.93.109.202:7777)
"That's a Sweet Machine.

Insert coin, recieve a sweet!"

Should be 'receive'.

### PR DESCRIPTION
**Pull request by DamianX**
Closes #13574
Closes #15489
Closes #16334
Closes #16408
Closes #16463
Closes #16978
Closes #16832
Closes #16758
Closes #9200
image

:cl:
 * spellcheck: Fixed several typos.
 * tweak: Made antag (cult/thrall/rev) (de)conversion messages bigger.

### CODE REVIEW

File: maps/RandomZLevels/hive.dm [hunk 1]
```diff
@@ -761,5 +761,5 @@ var/list/hive_pylons = list()
 //Rewards
 /obj/item/weapon/cloakingcloak/hive
 	name = "alien cloak"
-	desc = "Very light and soft to the tough, it's hard to believe that you would find something so delicate inside the Hive."
+	desc = "Soft to the tough."
```
Comment (Cyadox) [inline maps/RandomZLevels/hive.dm:764]: What did he mean by this :thinking:

Comment (MadmanMartian) [reply maps/RandomZLevels/hive.dm:764]: Is it not meant to be soft to the touch?

Comment (DamianX) [reply maps/RandomZLevels/hive.dm:764]: #16334

Comment (MadmanMartian) [reply maps/RandomZLevels/hive.dm:764]: pls no

**Comment by SonixApache** [comment]
whence

**Pull request merged**